conf/config.yaml

```diff
@@ -250,10 +250,10 @@ bridge:
     # See https://docs.mau.fi/bridges/general/end-to-bridge-encryption.html for more info.
     encryption:
         # Allow encryption, work in group chat rooms with e2ee enabled
-        allow: false
+        allow: true
```

Comment:
I may be wrong, but E2BE seems to me a rather advanced usage (because the homeserver is in most cases hosted on the same server as the bridge, without [Beeper](https://www.beeper.com/) nor with [delete_keys](https://docs.mau.fi/bridges/general/end-to-bridge-encryption.html?highlight=delete#additional-security) option set). If we want to support it, I'd suggest to make it optional:

https://docs.mau.fi/bridges/general/end-to-bridge-encryption.html?highlight=encryption#end-to-bridge-encryption

I am not excluding the relevance supporting it, I would suggest either:
 - not support it now and make it as an option later;
 - support it right now as an option (in the config panel);

Reply:
Probably in the administration (ADMIN.md) documentation, and we may also support toggling the `delete_keys ` option to be relevant.

If you want this feature, you may open an issue so we can implement it (right now or later, as you wish).

Reply:
It appeared you were right to assume that it could be required. A user had an error while trying to initiate a DM with the discord bridge bot account.

I cherry-pick your commit and this will probably be in the next version of the package.

Reply:
Yes, that's why I changed it, otherwise you have to change it manually.

Reply:
My bad!